websites/B/bs.to/presence.ts

```diff
@@ -9,7 +9,12 @@ presence.on("UpdateData", async () => {
 		largeImageKey: "bs",
 	};
 
-	if (document.location.hostname === "bs.to") {
+	if (document.location.hostname === "burningseries.domains") {
+		presenceData.startTimestamp = browsingTimestamp;
```

Comment:
Why not put this at the same place as largeImageKey?

Reply:
I don't know, i've just edited the presence to make the feature request. I've just found out that the timestamp is not active everytime, since it's probably a choice of the author, I've chosen to not touch it

Reply:
But I think a day the presence would need a codebase redoing